scripts/linters/pre_commit_linter.py

```diff
@@ -196,20 +196,13 @@ def _get_data(self, filepath, mode):
         return self._CACHE_DATA_DICT[key]
 
 
-def _lint_all_files(
-        js_filepaths, ts_filepaths, py_filepaths, html_filepaths,
-        css_filepaths, file_extensions_to_lint, verbose_mode_enabled=False):
+def _lint_all_files(file_extensions_to_lint, verbose_mode_enabled=False):
```

Comment:
are we expecting to receive a list of extensions here?

Reply:
no, just a single extension

Reply:
So why the arg name is file_extension**s**_to_lint (And same for the docstring)

Can you please update this method considering it to be a string not list.

Reply:
Done.